Gopkg.lock

```diff
@@ -2,777 +2,320 @@
 
 
 [[projects]]
-  digest = "1:1acadbbc24182315b628f727b2e9ac653266d1644ca4007e0766c28110afc072"
```

Comment:
Here is the problem and I don't want to redo this again (and my IDE does not want to let me revert this)